pkg/skaffold/output/output.go

```diff
@@ -52,7 +51,7 @@ func GetWriter(out io.Writer, defaultColor int, forceColors bool) io.Writer {
 	return skaffoldWriter{
 		MainWriter: SetupColors(out, defaultColor, forceColors),
 		// TODO(marlongamez): Replace this once event writer is implemented
-		EventWriter: ioutil.Discard,
+		EventWriter: eventV2.NewLogger(constants.DevLoop, "-1", "skaffold"),
```

Comment:
Do you want to define a constant for "topLevelIdx" or "IdxNone" and use it?

Reply:
will do 👍🏼